include/internal/catch_compiler_capabilities.h

```diff
@@ -235,7 +235,6 @@
   #    if defined(__clang__) && (__clang_major__ < 8)
          // work around clang bug with libstdc++ https://bugs.llvm.org/show_bug.cgi?id=31852
          // fix should be in clang 8, workaround in libstdc++ 8.2
-  #      include <ciso646>
```

Comment:
This needs to be kept in. The trick to this header is that it is basically free to include on any/all platforms, and is required to provide you with the version macros of the standard library (such as `__GLIBCXX__` used below).

In C++20, `<version>` has been standardized for this, but Catch2 v2 targets C++11 and v3 targets C++14.